File: blog-post-2/kotlin/Table.kt
import java.util.Scanner

fun main(args: Array<String>) {
        var scanner = Scanner(System.`in`)
        System.out.print("Enter a number for print table: ")
        var num = scanner.nextInt()
        var table : Int
        for(x in 1..10){
            table = num * x
            println(num.toString() + " X "+x+" = "+table)
        }
}

File: blog-post-2/cpp/Table.cpp
#include <iostream>
using namespace std;

int main()
{
    int num,table,i;
    cout << "Enter a number for print table: ";
    cin >> num;
    for (i = 1; i <= 10; i++)
    {
        table = num * i;
        cout << num <<" X "<< i << " = " << table << endl;
    }
}